cartography/intel/azure/util/common.py

```diff
@@ -5,7 +5,7 @@
 
 def parse_and_validate_azure_requested_syncs(azure_requested_syncs: str) -> List[str]:
     validated_resources: List[str] = []
-    for resource in azure_requested_syncs.split(','):
+    for resource in azure_requested_syncs:
```

Comment:
similar to aws & gcp, let's use the split method here to split comma separated services string